include/torch-mlir/Conversion/Passes.td

```diff
@@ -114,6 +114,32 @@ def ConvertTorchToTosa : Pass<"convert-torch-to-tosa", "func::FuncOp"> {
   let constructor = "mlir::torch::createConvertTorchToTosaPass()";
 }
 
+def ConvertTorchToTosaCustom : Pass<"convert-torch-to-tosa-custom", "func::FuncOp"> {
+        let summary = "Convert Torch ops to TOSA custom ops";
+  let description = [{
+    The purpose to use tosa::custom is handle complex ops when we donnot
```

Comment:
This is not a contract. A contract is something that describes in detail exactly what the expectations are before or after some transformation happens. For example, the backend contract in torch-mlir describes what the expectations are after conversion to torch backend is finished: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#the-backend-contract.

In this case, the contract should outline what each of the different torch dialect types will get turned into when this pass is finished. It should also outline exactly in what way attribute strings will be used (if they are used at all) to encode arguments. This document is meant to help users understand exactly what will happen when this pass takes place from just looking at the aten op signatures. Otherwise, there is no way for users to reliably know how to further lower `tosa::CustomOp` to their backend.

Reply:
Done